### DISCUSSION TITLE
new command to check if table exist

### DISCUSSION THREAD

**Discussion by lymaimei** [apache/doris#41294] [A - General / Q&A]
are we considering add new command (like exists) to check if a table is exist

**Comment by zy-kkk** [comment]
Doris can support adding the `IF EXISTS` keyword to the `Create` or `Drop` command, just like `DROP TABLE IF EXISTS your_table_name;` I don't know if this is what you mean. Secondly, Doris is compatible with MySQL. In my impression, MySQL does not seem to have a separate command to check whether a table exists. If you know, can you provide an example?

**Reply by lymaimei** [reply]
> Doris can support adding the `IF EXISTS` keyword to the `Create` or `Drop` command, just like `DROP TABLE IF EXISTS your_table_name;` I don't know if this is what you mean. Secondly, Doris is compatible with MySQL. In my impression, MySQL does not seem to have a separate command to check whether a table exists. If you know, can you provide an example?

thanks for your feedback
I found [show tables like 'x'] can be useful for my requirment
